app/models/cfe/remarks.rb

```diff
@@ -51,7 +51,7 @@ def populate_review_reasons
     end
 
     def populate_review_categories_by_reason
-      result = Hash.new([])
+      result = Hash.new([].freeze)
```

Comment:
I do not think this will be an issue as the default value frozen here is overwritten/assigned and therefore there be be no unassigned keys to mutate on subsequent assignments. 

What do others think?

Reply:
The syntax is weird, freeze then add data, but the documentation supports it and the tests pass... 🤷‍♂️